contents/ios/IntroToIos.md

```diff
@@ -0,0 +1,105 @@
+
+This also means that apps on the App Store are of higher quality in general. As iOS developers, we will be required to pay more attention to our application development process to make sure that everything is working and tested well before attempting to publish it up to the App Store. 
+
+## 3. Native iOS apps vs. cross-platform apps
+
+Mobile applications that run on the iOS platform can be written both natively and using [cross-platform solutions](http://www.businessofapps.com/guide/cross-platform-mobile-app-development/). Native iOS applications are written using [Objective-C or Swift](https://android.jlelse.eu/objective-c-or-swift-which-technology-to-learn-for-ios-app-development-3c681d1a05ac) on the Xcode IDE that you can download if you are running a MacOS. Cross-platform solutions are tools that allow you to write code once and develop applications for more than 1 platform. Some examples include [React Native](https://facebook.github.io/react-native/), [Xamarin](https://visualstudio.microsoft.com/xamarin/), and [Ionic](https://ionicframework.com/). Both methods have their own pros and cons, and these factors not only affect the app developers, but also the users to some extent.
+
+The main attraction to developing mobile applications using cross-platform solutions is development time. Cross-platform solutions allow you to write code once, but push it out to more than one OS, usually iOS and Android being two of these platforms. This means that essentially, your development time is cut by half, because you are only writing code once for 2 separate applications. While this may sound like an attractive deal, there are a bunch of caveats to consider that can potentially be a deal breaker, performance and compatability issues being the frontrunners.
+
+Some of these cross-platform solutions can be thought to work as an additional layer between your application and the native API calls, and with an additional layer comes inevitable performance compromise. This is most noticeable when it comes to more GPU demanding applications such as games. For less intensive applications such as social media apps, this is still manageable, as proven by Facebook and Instagram which are both written using React Native.
+
+Compatibility issues, especially with UI components, can come up when you are trying to develop a mobile app using cross-platform solutions, usually because some of these components are available on one OS but not the other, hence the cross-platform API will completely not give the component as an option to developers. 
+
+In a nutshell, there are many factors to consider when deciding to develop mobile apps using cross-platform or native solutions, but as budding software engineers, it definitely would not hurt to pick up both!
```

Comment:
Which is easier to pick up for starters?

Reply:
It's quite hard to give a definitive answer to this... I'll add a few more points to about ease of learning cross-platform vs. native